prow/istio-pilot-multicluster-e2e.sh

```diff
@@ -16,10 +16,10 @@
 
 # set this to the new multicluster-e2e type
 export RESOURCE_TYPE="${RESOURCE_TYPE:-gke-e2e-test}"
-export OWNER=istio-pilot-multicluster-e2e
+export OWNER="istio-pilot-multicluster-e2e"
 
 export SETUP_CLUSTERREG="True"
-CLUSTERREG_DIR=${CLUSTERREG_DIR:-$(mktemp -d /tmp/clusterregXXX)}
+CLUSTERREG_DIR="${CLUSTERREG_DIR:-$(mktemp -d /tmp/clusterregXXX)}"
```

Comment:
might consider more xxx - my understanding is prow is a shared machine and you may run into collisions with such a small address space.

Reply:
this is running in a container spawned every run by prow.  IMO, it's safe as is.

Reply:
@tiswanso  is right. Every job run in a container.